src/components/PostDetails.tsx

```diff
@@ -157,9 +121,7 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
           </button>
         )}
 
-        {loaded && !hasError && visible && (
-          <NewCommentForm onSubmit={addComment} />
-        )}
+        {loading && !err && visible && <NewCommentForm onSubmit={addComment} />}
```

Comment:
The condition `loading && !err && visible` might not be correct for displaying the `NewCommentForm`. Typically, you would want to display the form after loading is complete, i.e., `!loading && !err && visible`.